src/Data/List/Relation/Unary/First/Properties.agda

```diff
@@ -59,6 +59,12 @@ module _ {a p q} {A : Set a} {P : Pred A p} {Q : Pred A q} where
   First⇒¬All q⇒¬p [ qx ]     (px ∷ pxs) = q⇒¬p qx px
   First⇒¬All q⇒¬p (_ ∷ pqxs) (_ ∷ pxs)  = First⇒¬All q⇒¬p pqxs pxs
 
+  ¬First⇒All : ∁ Q ⊆ P → ∀ xs → ∁ (First P Q) xs → All P xs
```

Comment:
Why not `∁ (First P Q) ⊆ All P`?

Reply:
My idea was to make `¬First⇒All`'s signature close to `¬Any⇒All¬`'s where the argument `xs` is explicit. Using `P Relation.Unary.⊆ Q`, it will be implicit on the other hand.
But, I just found `Relation.Unary._⊆′_`, the explicit version of `_⊆_`. I should have used it instead.

As mentioned in the first comment, I am not sure whether to make the argument `xs` explicit or implicit.
In `¬Any⇒All¬` it is explicit, but why is it?

Reply:
I am not sticking to having it explicit, so I will update the signature to `¬First⇒All : ∁ Q ⊆ P → ∁ (First P Q) ⊆ All P` then.